### ISSUE TITLE
Killaura Improvements

### ISSUE THREAD

**Issue by Jakaboii** [joehot200/AntiAura#9]
There's some "drama" (Don't know what else to call it) about the KillAura check

Some say it's easy to bypass, some say it's okay and when I tested it, it wasn't really the best & I could bypass it

It'd be nice if the KillAura check got improved & maybe could be configurable in the config

Just a suggestion, and since it's called AntiAura and doesn't actually have configurable settings (that I could find by searching "kill", "aura" & "killaura") it's kinda ironic

**Comment by joehot200** [comment]
The settings are under "forcefield", which was the old term for Killaura. There goes AntiAura showing its old age :D

**Comment by Jakaboii** [comment]
Oh, OK

I'll make that stricter and see if that makes much of a difference

**Comment by Jakaboii** [comment]
I changed loads of stuff and made it very strict but it still doesn't detect very blatant KillAura

The way that it "takes a while" to detect, in my opinion, isn't very efficient or effective 

As most of the time you don't really get super long pvp fights in KitPvP, Practice or anything really (Unless both of the people use god apples, but that doesn't always happen either)

Hopefully you can make it more efficient and quicker at detecting KillAura when you fix the obfuscation (I didn't think it'd really effect premium plugins, but I guess Spigot really messed up here)

**Comment by joehot200** [comment]
@Jakaboii not to worry, I've put it at mid priority and will be taking a look.

**Comment by Jakaboii** [comment]
Alright, thanks a lot!

**Comment by Jakaboii** [comment]
I changed some combat settings, it seems to detect faster now & doesn't false positive

I did have an issue with badlion client though, sometimes the Accuracy check falsed and kicked/banned me when I died

**Comment by joehot200** [comment]
> I changed some combat settings, it seems to detect faster now & doesn't false positive
> 
> I did have an issue with badlion client though, sometimes the Accuracy check falsed and kicked/banned me when I died

What did you change? Might just help me out when I improve the killaura checks.

**Comment by Jakaboii** [comment]
Here's a [comparison website](https://www.diffchecker.com/cQSvjCyV), this just has the ForceField & FastClick settings in it, I changed a few other settings though I'm pretty sure so if you need those then just ask

Both files in the comparison website are on the latest version

I just did brief testing, but it detected KillAura way quicker than it did beforehand

Something to note though is I did the first bit of KillAura testing on my main account (which for some reason doesn't get flagged, may be a LuckPerms issue) so after I figured out my account didn't get flagged, I moved over to my alt account with the new configuration

So basically, I haven't tested the default KillAura config because it didn't flag my main account & I didn't test the default one on my alt account

I'm really bad at explaining things hopefully it makes a bit of sense

**Comment by dodofan** [comment]
Killaura improvements seem like they will come in 11.88. Not everything you wanted, of course, but I think a lot of what you wanted will be done.
